ETC.tsv

```diff
@@ -3811,7 +3811,7 @@ ETC_20150317_003810	Quest Starting Map:
 ETC_20150317_003811	Start NPC:
 ETC_20150317_003812	$ activated {nl}
 ETC_20150317_003813	 Use this to warp anywhere!
-ETC_20150317_003814	This camp does not have an activated Goddess Statue.{nl} Therefore you cannot move.
+ETC_20150317_003814	{memo Check if this appears in any other scenario. Don't put a $ here, changes linebreak.}You have not activated any other Goddess Statue, so you cant teleport yet.
```

Comment:
The original is fine, maybe change it to: "You cannot move/warp because this camp does not have an activated Goddess Statue."

As fas as I know, this is only in regards to the tutorial area.